<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Retype Password: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Register2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="Date_of_Birth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>January</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>February</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>March</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>April</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>May</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>June</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>July</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>August</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>September</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>October</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>November</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>December</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Gender</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Account Type</string>
   </property>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Comedy</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Romance</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drama</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horror</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
